react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: '',
+      isValid: false,
+      errorMessage: 'Please enter a password'
+    };
+
+    this.tooLittleChar = this.tooLittleChar.bind(this);
+  }
```

Comment:
It doesn't look like you're using `isValid` or `errorMessage` in state, so I'd suggest removing them.